docs/templates/template_reference.md

```diff
@@ -815,9 +815,26 @@ The selected value can be changed in the profile (consult the actual variable fo
 
     -   **datatype** - data type of the sysctl value, eg. `int`.
 
-    -   **sysctlval** - value of the sysctl value, eg. `'1'`. If this
+    -   **sysctlval** - value of the sysctl value. This can be either an atomic
+        value, eg. `'1'`, or a list of values, eg. `['1','2']`. If this
         parameter is not specified, XCCDF Value is used instead.
 
+    -   **sysctlval_remediate** - the value that will be used in remediations.
```

Comment:
This could be replaced by an XCCDF variable. The variable would be only used in the remediation and would allow users to tailor the value, while here the value is hardcoded and can be only changed during build time.